packages/qwik-city/package.json

```diff
@@ -97,9 +97,19 @@
       "require": "./lib/adapters/static/vite/index.cjs"
     },
     "./adapters/vercel-edge/vite": {
-      "types": "./lib/adapters/vercel-edge/vite/index.d.ts",
-      "import": "./lib/adapters/vercel-edge/vite/index.mjs",
-      "require": "./lib/adapters/vercel-edge/vite/index.cjs"
+      "types": "./lib/adapters/vercel/edge/index.d.ts",
+      "import": "./lib/adapters/vercel/edge/index.mjs",
+      "require": "./lib/adapters/vercel/edge/index.cjs"
+    },
+    "./adapters/vercel/edge": {
+      "types": "./lib/adapters/vercel/edge/index.d.ts",
+      "import": "./lib/adapters/vercel/edge/index.mjs",
+      "require": "./lib/adapters/vercel/edge/index.cjs"
+    },
+    "./adapters/vercel/serverless": {
```

Comment:
All the other adapters are xx/vite, so I think you should keep that naming scheme.

Reply:
Thanks, done 👍🏽